features/admin.applications.v1/pages/applications-settings.tsx

```diff
@@ -0,0 +1,430 @@
+                        updateConfigurations(values);
+                    } }
+                    initialValues={ {
+                        ssaJwks: ssaJwksState,
+                        authenticationRequired: isAuthenticationRequired,
+                        mandateSSA: isMandateSSA,
+                        enableFapiEnforcement: isEnableFapiEnforcement,
+                        dcrEndpoint: dcrEndpointState
+
+                    } }
+                    validate={ validateForm }
+                    data-componentid={ `${ componentId }-form` }
+                >
+                
+                    <Field.Input
+                        ariaLabel="DCR Endpoint"
+                        inputType="text"
+                        name="dcrEndpoint"
+                        label={ t("applications:forms.applicationsSettings.fields.dcrEndpoint.label") }
+                        placeholder={ t("applications:forms.applicationsSettings.fields.dcrEndpoint.placeholder") }
+                        hint={ t("applications:forms.applicationsSettings.fields.dcrEndpoint.hint") }
+                        required={ false }
+                        readOnly={ true }
+                        maxLength={ 150 }
+                        minLength={ 10 }
+                        width={ 16 }
+                        listen={ null }
+                        data-componentid={ `${componentId}-dcr-endpoint-url` }
+                    >
+                        <CopyInputField
+                            value={ dcrEndpointState }
+                        />
+                    </Field.Input>
+                    <Field.Checkbox
+                        ariaLabel="Require Authentication"
+                        name="authenticationRequired"
+                        label={ t("applications:forms.applicationsSettings.fields.authenticationRequired.label") }
+                        hint={ (
+                            <>
+                                { t("applications:forms.applicationsSettings.fields.authenticationRequired.hint") }
+                                <DocumentationLink
+                                    link={ getLink("develop.applications.applicationsSettings.dcr.authenticationRequired.learnMore") }
+                                >
+                                    { t("console:develop.pages.applicationsSettings.learnMore") }
+                                </DocumentationLink>
+                            </>
+                        ) }
+                        tabIndex={ 3 }
+                        width={ 16 }
+                        listen={ 
+                            function (value: boolean) {
+                                setAuthenticationRequired(value);
+                                if (!value) {
+                                    setMandateSSA(true);
+                                }
+                            }
+                        }
```

Comment:
Please use the arrow function here to preserve the coding consistency.

Reply:
Fixed with [d54ec35](https://github.com/wso2/identity-apps/pull/5964/commits/d54ec35c7ee212f1039c9afea0052b20884b03c3)